src/llv8-inl.h

```diff
@@ -89,7 +89,7 @@ inline int64_t Map::GetType(Error& err) {
       v8()->LoadUnsigned(LeaField(v8()->map()->kInstanceAttrsOffset), 2, err);
   if (err.Fail()) return -1;
 
-  return type & 0xff;
+  return type;
```

Comment:
Hmm, so the CI is failing because on older versions of V8 this is a uint8_t, while on v6.4 this is uint16_t...should probably set a flag or something during the constant loading so we can be sure which part we should mask off here?

Reply:
That sounds like a good idea. I _think_ we conditionally set a mask somewhere else, but I could be mistaken.